data/remnant/remnant missions.txt

```diff
@@ -4479,3 +4479,140 @@ mission "Remnant: Expanded Horizons Astral job"
+		conversation
+			`"Thank you for your help, <first>. Sometimes I have to wait a while before we have a ship available to do missions like these." She pauses, then taps her commlink. "Speaking of which, I have confirmed the transfer of the logistics fee of <payment> to your account."`
+			choice
+				`	"I am always happy to help. What are you working on now, anyway?"`
+				`	"I am always happy to help. See you around!"`
+					goto stormend
+			`	"How familiar are you with the nature of space?"`
+			choice
+				`	"Not very, to be honest."`
+				`	"I know space is a vacuum, and can be either extremely hot or extremely cold depending on circumstances..."`
+					goto basics
+			`	She nods "Honesty is good. It is hard to learn when we are not willing to admit that we do not know something."`
+				goto explanation
+			label basics
+			`	She nods "Those are the basics, a good starting point."`
+			label explanation
+			`	A hologram of the Ember Waste appears over her commlink. "As you have no doubt picked up on, the Ember Waste does not conform to humanity's commonly understood rules about space. For instance, the standard background temperature is 2.B1 degrees, and the average density of the interstellar medium is F4240 particles per cubic meter. Here in the Ember Waste, the average background temperature is an order of magnitude greater and has an average density of E8D4A51000 particles per cubic meter."`
+			`	The display changes as an overlay of haze appears, pulsing in intensity. "Most interestingly, though, is that this area is significantly more variable than the rest of the galaxy. The haze shown here represents the heat hazard schematic we produce for our captains. It combines the local density with the measured heat values to estimate how much heat ships will be exposed to in the area."`
+			choice
+				`	"Interesting! Is that what causes the red glow?"`
+				`	"Wait a second. Why are there letters in your numbers?"`
+					goto numericsystem
+				`	"Thanks! I think that is all the details I can handle right now."`
+					goto stormend
+			label stormresume
+			`	"Not really, although it does contribute. Most of the red glow seems to be the result of clouds of hydrogen gases at aproximately 2710 degrees, rather like the process documented by early Earth astronomers investigating nebulae. Except for most of these clouds, there is no obvious source of energy to keep the hydrogen excited. Obviously there is still energy being added to the system, but..." She trails off and gestures depreciatingly. "One more mystery to keep us alert and attentive."`
+			`	She smiles thoughtfully, and the display changes with the red haze vanishing and a silvery haze replacing it. "But that is where today's research comes in. We have been measuring these storms since we arrived in the Ember Waste. The human eye can only pick out a few of them on the display, but our algorythms have identified a significant number of ripples of energy." She appears thoughtful.`
```

Comment:
```suggestion
			`	She smiles thoughtfully, and the display changes with the red haze vanishing and a silvery haze replacing it. "But that is where today's research comes in. We have been measuring these storms since we arrived in the Ember Waste. The human eye can only pick out a few of them on the display, but our algorithms have identified a significant number of ripples of energy." She appears thoughtful.`
```